k8sclient/k8sclient_test.go

```diff
@@ -567,6 +567,113 @@ var _ = Describe("k8sclient operations", func() {
 		Expect(netConf.Delegates[0].Conf.Type).To(Equal("mynet1"))
 	})
 
+	It("adds extra data to the default network", func() {
```

Comment:
Could you please add following tests to cover them?

- "v1.multus-cni.io/default-network" has json net-attach-def